### PR TITLE
Update docs with getting help and add Cython dependencies

### PR DESCRIPTION
**Pull request by bocklund**
I'm not sure if the documentation commit should be here to automatically trigger the rebuild of the website. Should I just move the DOC commit over to my website branch and submit the PR to the pycalphad/website branch?

### CODE REVIEW

**Comment by richardotis** [comment]
I think you did this correctly. The deployment script should automatically commit to the website branch after this commit is merged to develop.

File: docs/index.rst [hunk 1]
```diff
@@ -37,6 +35,17 @@ A public chat channel is available on `Gitter`_.
...
 
+Getting Help
+============
+
+Questions about installing and using pycalphad can be addressed in the `pycalphad Google Group`_.
+Technical issues and bugs should be reported on on `GitHub`_.
+A public chat channel is available on `Gitter`_.
+The support email list can be reached at support@pycalphad.org.
+
+.. _Gitter: https://gitter.im/richardotis/pycalphad
```
Comment (richardotis) [inline docs/index.rst]: This should be https://gitter.im/pycalphad/pycalphad

File: docs/index.rst [hunk 2]
```diff
@@ -37,6 +35,17 @@ A public chat channel is available on `Gitter`_.
    api/modules
    CHANGES
 
+Getting Help
+============
+
+Questions about installing and using pycalphad can be addressed in the `pycalphad Google Group`_.
+Technical issues and bugs should be reported on on `GitHub`_.
+A public chat channel is available on `Gitter`_.
+The support email list can be reached at support@pycalphad.org.
```
Comment (richardotis) [inline docs/index.rst]: We cannot merge this until I've actually done something for email handling.

**Comment by bocklund** [comment]
Fixed the link for gitter. I also changed the GitHub links to the pycalphad/pycalphad repo for clarity.
I left the email link in so we can merge whenever that is set up. I can remove it if you want to merge in the Cython dependencies now.

**Comment by richardotis** [comment]
I'm +1 to removing the email link and merging now. The Google group will meet most users' needs, and those who want more privacy can find our respective email addresses without much effort.

**Pull request closed without merging**